styles/globals.css

```diff
@@ -176,3 +176,7 @@ a:active {
     font-size: 24px;
   }
 }
+
+.MuiFormLabel-root {
+  color: black !important;
```

Comment:
This might mess with your validation, just be careful

Reply:
Interesting I did not realise I had made any changes to global.css on this branch. The form seems fine and the colour is correct. I am going to merge and will be sure to double check validation before deployment.